### ISSUE TITLE
PackagePublisher - InvalidOperationException: Collection was modified

### ISSUE THREAD

**Issue by mikeharder** [aspnet/Coherence#177]
PackagePublisher failed with the following error in build Coherence#24169:

```
System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
```

**Comment by pranavkm** [comment]
Might be a dup of https://github.com/aspnet/BuildTools/issues/221.

**Comment by mikeharder** [comment]
Might be the same root cause, but two different source files.

**Comment by kichalla** [comment]
Fixed with 2b71b3d0a83689ebdf8e6ee2ad0ee3349cd0ba22
